sample/sample/sample.cpp

```diff
@@ -0,0 +1,86 @@
+// sample.cpp : Defines the entry point for the console application.
+//
+
```

Comment:
As opposed to C# in C++ async any time a task may throw (which in case of SignalR C++ Client is true any public method returning a task) you need to observe exceptions. If you don't the unobserved exception will crash your process.